src/llm_tracekit/bedrock/invoke_agent.py

```diff
@@ -61,22 +146,42 @@ def generate_attributes_from_invoke_agent_input(
 
 
 def record_invoke_agent_result_attributes(
-    content: Optional[str],
-    usage_input_tokens: Optional[int],
-    usage_output_tokens: Optional[int],
+    result: AgentStreamResult,
     span: Span,
     start_time: float,
     instruments: Instruments,
     capture_content: bool,
 ):
     try:
+        current_choice = Choice(
+            role="assistant",
+            content=result.content,
+            tool_calls=result.tool_calls_buffer or None,
+        )
+
+        if result.chat_history:
```

Comment:
`if result.chat_history is not None` - fix this for all conditions on nullable fields